introduction.md

```diff
@@ -1,2 +1,6 @@
 hi I am maxi and I'am trying mastering git :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?